phantom-dsl/src/main/scala/com/websudos/phantom/builder/query/prepared/PreparedUpdateQuery.scala

```diff
@@ -163,16 +173,32 @@ sealed class AssignmentsQuery[
     new ConditionalQuery(table, init, usingPart, wherePart, setPart, casPart append query)
   }
 
-  def consistencyLevel_=(level: ConsistencyLevel)(implicit ev: Status =:= Unspecified): AssignmentsQuery[Table, Record, Limit, Order, Specified, Chain] = {
-    new AssignmentsQuery(
-      table,
-      init,
-      usingPart append QueryBuilder.consistencyLevel(level.toString),
-      wherePart,
-      setPart,
-      casPart
-    )
+  def consistencyLevel_=(level: ConsistencyLevel)(implicit ev: Status =:= Unspecified, session: Session): AssignmentsQuery[Table, Record, Limit, Order, Specified, Chain] = {
```

Comment:
The only information required from session is the protocol version, therefore it'd be cleaner just to pass this as argument. Implicit conversion should help with avoiding boiler plate required to extract it from session